src/wasm-lib/kcl/src/coredump/mod.rs

```diff
@@ -74,18 +85,44 @@ pub trait CoreDump: Clone {
             webrtc_stats,
             github_issue_url: None,
             pool: self.pool()?,
+            client_state,
         };
-        app_info.set_github_issue_url(&screenshot_url)?;
 
-        Ok(app_info)
+        // pretty-printed JSON byte vector of the coredump.
+        let data = serde_json::to_vec_pretty(&core_dump_info)?;
+
+        // Upload the coredump.
+        let links = zoo_client
+            .meta()
+            .create_debug_uploads(vec![kittycad::types::multipart::Attachment {
+                name: "".to_string(),
+                filename: Some(format!(r#"modeling-app/coredump-{}.json"#, coredump_id)),
+                content_type: Some("application/json".to_string()),
+                data,
+            }])
+            .await
+            .map_err(|e| anyhow::anyhow!(e.to_string()))?;
+
+        if links.is_empty() {
+            anyhow::bail!("Failed to upload coredump");
+        }
+
+        let coredump_url = links[0].clone();
```

Comment:
No need to clone this, because you're just borrowing `coredump_url` a few lines down anyway.
```suggestion
        let coredump_url = &links[0];
```